fluent-syntax/src/ftlstream.js

```diff
@@ -128,6 +128,39 @@ export class FTLParserStream extends ParserStream {
     return false;
   }
 
+  // -1 - any
+  //  0 - comment
+  //  1 - group comment
+  //  2 - resource comment
+  isPeekNextLineComment(level = -1) {
+    if (!this.currentPeekIs('\n')) {
+      return false;
+    }
+
+    let ptr = 0;
```

Comment:
In other parts of this file, `ptr` is used to store a peek index. Perhaps rename this simply to `i`?